.gitignore

```diff
@@ -58,3 +58,16 @@ callgrind.out.*
 # Miscellaneous
 *.log
 /.gtm/
+
+# Ignore all bazel-* symlinks. There is no full list since this can change
+# based on the name of the directory bazel is cloned into.
+/bazel-*
+MODULE.bazel.lock
+.bazelrc.user
+
```

Comment:
hedron compile commands adds an external folder which you can also ignore if you want

Reply:
Good to know! Do you know which one?

Reply:
the folder is called „external“ 😆

Reply:
Thanks!